src/stavenote.ts

```diff
@@ -426,6 +427,12 @@ export class StaveNote extends StemmableNote {
     }
     this.reset();
     this.buildFlag();
+    if (this.noteType === 'p') {
```

Comment:
Can we move this to `Factory` please? -- it doesn't need to be a `noteType`, otherwise we'll end up with a lot of special cases. See my comment above re: `build(targetNote)`.

Reply:
Done